src/schemas/rinkeby/rinkebyENSShortNameAuction/index.ts

```diff
@@ -0,0 +1,86 @@
+import { AbiType } from 'web3';
+
+import {
+  ENSName,
+  ENSNameBaseSchema,
+  namehash,
+  nodehash,
+} from '../../../common/ens';
+import {
+  EventInputKind,
+  FunctionInputKind,
+  Schema,
+  StateMutability,
+} from '../../../types';
+
+export const RINKEBY_ENS_SHORT_NAME_AUCTION_ADDRESS =
+  '0x266ebd3ca5b8c472da228df3f7ab2cc951b19ec3';
+
+export const rinkebyENSShortNameAuctionSchema: Schema<ENSName> = {
```

Comment:
Any reason we're naming this `rinkebyENSShortNameAuctionSchema` instead of `ENSShortNameAuctionSchema`? Wouldn't we want to use the same schema in prod?

Reply:
The `RINKEBY_ENS_SHORT_NAME_AUCTION_ADDRESS` and block numbers are different. When the contract is deployed on mainnet I'll add the mainnet copy. I noticed that we have rinkeby vs mainnet folders, so that's why I'm doing it this way.
I think we can think about refactoring all the commonalities of wyvern schemas, but we don't add schemas very often anyway.